benchmark/attention_backend/bench.py

```diff
@@ -0,0 +1,242 @@
+import argparse
+import itertools
+import re
+import subprocess
+import threading
+from collections import defaultdict
+from dataclasses import dataclass
+from io import StringIO
+from pathlib import Path
+from typing import List, Tuple
+
+import torch
+import yaml
+
+from sglang.utils import launch_server_cmd, terminate_process, wait_for_server
```

Comment:
Typically, we avoid introducing dependencies related to sglang in order to make the script more standalone and usable without installing sglang.